packages/react-components/src/components/SvgExclamationIcon/SvgExclamationIcon.tsx

```diff
@@ -0,0 +1,19 @@
+// This icon is available as a plain SVG at src/assets/icon-exclamation.svg
+export default function SvgExclamationIcon({ id = "Exclamation icon" }) {
```

Comment:
Sorry, just caught this - it's not valid to use whitespace in an HTML attribute (even if it works in most browsers), so this `id` prop default should become `exclamation-icon`.

Reply:
Fixed in [cbe0370](https://github.com/bcgov/design-system/pull/400/commits/cbe0370c75e09322f59be838a08390d84fdc9cc9)